inspire/modules/workflows/tasks/submission.py

```diff
@@ -91,6 +91,7 @@ def _create_ticket(obj, eng):
             email=email,
             title=d.title,
             identifier=sip.metadata.get("system_number_external", {}).get("value", ""),
+            user_comment=sip.metadata.get("hidden_note", {}).get("value", ""),
```

Comment:
what happens if we add a couple of hidden notes? Then:

```python
sip.metadata.get("hidden_note", {})
```

will be a list right?

We should probably take that into account and also just get the hidden note that has ``source: submitter`` - see https://github.com/inspirehep/inspire-next/pull/192